Project1/DriverLicenceReg.py

```diff
@@ -0,0 +1,79 @@
+def is_date_valid(date):
+	correctDate = None
+	try:
+		this_date = datetime.strptime(date, '%m-%d-%Y')
+		correctDate = True
+	except ValueError:
+		correctDate = False
+	return correctDate
+		
+def DriverLicenceReg():
+	print("Enter 'menu' to go back to main menu.")
+	print("Enter 'quit' to quit from the system.")
+	print("Enter 'insert' to register new driver licence.")
+	while(True):
+		Input=input("-->")
+		if( Input == "quit"):
+			print("Exitting...")
+			break
+			#sys.exit()
+		if( Input == "menu"):
+			MainMenu()
+		if( Input == "insert"):
+			licence_no = input("Please enter Licence Number:")
+			while( len(licence_no)>15 or license_no == ""): #If licence no has more than 15 character and does not entered anything
+				print("The licence number that you entered is invalid. Please try again.")
+				licence_no = input("Please enter Licence Number:")
+			sin = input("Please enter Social Insurance Number:")
+                        while( len(sin) > 15 or sin == "" ):
+                                print("The Social Insurance Number that you entered is invalid. Please try again.")
+                                licence_no = input("Please enter Social Insurance Number:")
+			licence_class = input("Please enter Licence Class:")
+			while ( len(licence_class)>10) or licence_no = ""):
+				print ("The Licence Class that you entered is invalid. Please try again.")
+				licence_class = input("Please enter Licence Class:")
+			photo_name = input("Please insert photo for licence (Optional) :")
+			issuing_date = ("Please enter issuing date of the licence in MM-DD-YYYY format:")
+			while ( is_date_valid(issuing_date) == False): #I need to check if it is in MM-DD-YYYY format
+				print ("Issuing Date that you entered is invalid. Please try again.")
```

Comment:
I forgot to put this statement "issuing_date = ("Please enter issuing date of the licence in MM-DD-YYYY format:")"  after printing the date is invalid. I need to put the similar line for expiring_date as well